ibis/backends/risingwave/__init__.py

```diff
@@ -257,3 +311,364 @@ def _register_in_memory_table(self, op: ops.InMemoryTable) -> None:
+        schema
+            Name of the schema where the view exists, if not the default.
+        """
+        if database is not None and database != self.current_database:
+            raise com.UnsupportedOperationError(
+                f"Creating sinks in other databases is not supported by {self.name}"
+            )
+        else:
+            database = None
+
+        table = sg.table(name, db=schema, catalog=database, quoted=self.compiler.quoted)
+
+        if encode_format is None != data_format is None:
+            raise com.RelationError(
+                "When creating sinks, both encode_format and data_format must be provided, or neither should be"
+            )
+
+        if sink_from is not None:
+            create_stmt = f"CREATE SINK {table.sql(self.dialect)} FROM {sink_from}"
+        else:
+            create_stmt = sge.Create(
+                this=table,
+                kind="SINK",
+                expression=self.compile(obj),
+            ).sql(self.dialect)
+        create_stmt = (
+            create_stmt
+            + " WITH "
+            + self._format_properties(connector_props)
+            + self._data_and_encode_format(
+                data_format, encode_format, encode_properties
+            )
+        )
+        with self._safe_raw_sql(create_stmt):
+            pass
+
+    def drop_sink(
+        self,
+        name: str,
+        *,
+        database: str | None = None,
+        schema: str | None = None,
+        force: bool = False,
+    ) -> None:
+        """Drop a Sink.
+
+        Parameters
+        ----------
+        name
+            Sink name to drop.
+        database
+            Name of the database where the view exists, if not the default.
+        schema
+            Name of the schema where the view exists, if not the default.
+        force
+            If `False`, an exception is raised if the source does not exist.
+        """
+        src = sge.Drop(
+            this=sg.table(
+                name, db=schema, catalog=database, quoted=self.compiler.quoted
+            ),
+            kind="SINK",
+            exists=force,
+        )
+        with self._safe_raw_sql(src):
+            pass
+
+    def _data_and_encode_format(self, data_format, encode_format, encode_properties):
+        res = ""
+        if data_format is not None:
+            res = res + " FORMAT " + data_format.name
+        if encode_format is not None:
+            res = (
+                res
+                + " ENCODE "
+                + encode_format.name
+                + self._encode_properties_to_str(encode_properties)
+            )
+        return res
+
+    def _encode_properties_to_str(self, encode_properties):
+        return (
+            ""
+            if encode_properties is None
+            else " " + self._format_properties(encode_properties)
+        )
+
+    def _format_properties(self, props):
+        tokens = []
+        for k, v in sorted(props.items()):
```

Comment:
Do these need to be sorted?  If not, I think we should render them in the order they were provided.

Reply:
`sorted` removed